src/curated-corpus/components/ExistingProspectCard/ExistingProspectCard.test.tsx

```diff
@@ -93,7 +93,7 @@ describe('The ExistingProspectCard component', () => {
     expect(screen.getByText(/^de$/i)).toBeInTheDocument();
   });
 
-  it('shows topic correctly', () => {
+  it('should show topic correctly', () => {
```

Comment:
Took this opportunity to review the unit tests for both components and adjust wording/improve the tests where possible.